components/global/drupal-form--default.vue

```diff
@@ -22,3 +22,52 @@ defineSlots<{
   default();
 }>()
 </script>
+
+<style lang="postcss">
+.drupal-form {
+  @apply font-meta text-foreground;
+
+  /* Input fields */
+  input:not([type="checkbox"]):not([type="radio"]),
+  select,
+  textarea {
+    @apply w-full px-4 py-2 rounded-md;
+    @apply border-input bg-background text-foreground;
+    @apply focus:outline-none focus:ring-2 focus:ring-ring focus:ring-offset-2;
+  }
+
+  label {
+    @apply block text-sm font-medium text-foreground mb-1;
+  }
+
+  .form-item {
+    @apply mb-4;
+  }
+
+  /* Buttons */
+  button[type="submit"],
+  input[type="submit"] {
+    @apply w-full px-4 py-2 text-sm font-medium;
```

Comment:
not sure why we need those drupal-form customizations, could we just configure them as global defaults?